content/blog/bun-package-manager/index.md

```diff
@@ -0,0 +1,319 @@
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2025-07-28T09:50:26+01:00
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Learn how to use Bun as a package manager with Pulumi for faster dependency management. Get faster installations with step-by-step setup guides.
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - piers-karsenbarg
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - typescript
+    - package-manager
+
+---
+
+Today, we are announcing a new addition that allows Pulumi users to use [Bun](https://bun.sh/) as their package manager, offering an alternative to npm, Yarn, or pnpm. Bun, a recently released JavaScript runtime, bundler, transpiler, and package manager, provides a faster way to manage Pulumi project dependencies.
+
+<!--more-->
+
+## What is this Bun thing you're talking about?
+
+If you've spent time around the JavaScript ecosystem, then you've probably heard of Bun. If you haven’t, then be aware that it presents itself as a JavaScript toolkit, a direct drop-in replacement for NodeJS. However, the part that we like at Pulumi is its use as a package manager. According to the [Bun install documentation](https://bun.com/docs/cli/install), switching from `npm install` to `bun install` can make your installations up to 25 times faster.
+
+{{% notes type="warning" %}}
+
+1. Pulumi uses Bun solely as a package manager. You cannot use Bun to write Automation API-based applications, nor do we have a language plugin for Bun.  
+2. This requires Bun version 1.2 or greater due to the text-based lockfile introduced in this version.
+
+{{% /notes %}}
+
+## Getting started with Bun and Pulumi
+
+Since you're probably already using NodeJS with Pulumi to want to use this, the easiest way to install Bun is to run the following:
```

Comment:
If find this sentence awkward due to the clause at the beginning.
Maybe change the paragraph to something like:

```
The assumption is that you are already using NodeJS with Pulumi.
So, if you want to try Bun with Pulumi, the easiest way to install Bun is to run the following: ....
```